packages/trpc/server/routers/viewer/availability/schedule/getScheduleByUserId.handler.ts

```diff
@@ -0,0 +1,56 @@
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../../trpc";
+import { getHandler } from "./get.handler";
+import type { TGetByUserIdInputSchema } from "./getScheduleByUserId.schema";
+
+type GetOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TGetByUserIdInputSchema;
+};
+
+const EMPTY_SCHEDULE = [[], [], [], [], [], [], []];
```

Comment:
Empty schedule if no default availability found  (Means they have not completed onboarding)

Reply:
i wonder if an admin can pre-fill availability for someone before they finish onboarding?